src/constants/graphqlQueries.ts

```diff
@@ -26,12 +26,18 @@ export const generateL2RasterProductQuery = `
         }
     }
 `
+// , $cycle: Int, $pass: Int, $scene: Int, $outputGranuleExtentFlag: Boolean, $outputSamplingGridType: GridType, $beforeTimestamp: String, $afterTimestamp: String
+// , cycle: $cycle, pass: $pass, scene: $scene, outputGranuleExtentFlag: $outputGranuleExtentFlag, outputSamplingGridType: $outputSamplingGridType, beforeTimestamp: $beforeTimestamp, afterTimestamp: $afterTimestamp
```

Comment:
Don't comment out code - just remove it. We can always look it back up within the commit history

Reply:
Got it, this was to help me copy past something and I forgot to remove it